build_defs/java.build_defs

```diff
@@ -87,6 +90,33 @@ def java_library(name:str, srcs:list=None, src_dir:str=None, resources:list=[],
 
         src_dir_label = [] if srcs else ['src_dir:'+src_dir]
 
+        def _jni_header_outputs(name, output):
+            for hdr_path in output:
+                if hdr_path != "":
```

Comment:
This is just to guard against the command outputting `\n` when no headers are generated.